mlflow/R/mlflow/R/cli.R

```diff
@@ -40,6 +29,11 @@ mlflow_cli <- function(...,
     MLFLOW_CONDA_HOME = python_conda_home(),
     MLFLOW_TRACKING_URI = mlflow_get_tracking_uri()
   ), env)
+  if (is.null(stderr_callback)) {
```

Comment:
By default, print stderr from invoking CLI commands. This is pretty helpful for debugging issues that arise when invoking Python CLI commands from R / will help users give us useful diagnostic info if/when anything breaks.